### DISCUSSION TITLE
Financial Support

### DISCUSSION THREAD

**Discussion by Lockyc** [loco-rs/loco#108] [General]
Hi, Loco looks excellent!
And the potential is awesome!

Is there some way we can financially support the project?

**Comment by jondot** [comment]
Hi!
So excited you like this project!!
Right now we do not require any financial support, this is a project that we're doing in our free time, and most of the services we use are free (github, hosting, etc.)

What we would love to accept is more code contribution, help with the docs, with code, and anything else -- so feel free to jump in, or if you don't have the time -- we would love a mention on social networks and friends that would be interested to contribute :)

**Comment by Lockyc** [comment]
Thanks for your reply jondot
I knew I could make contributions in all those ways. 
And I would contribute in those ways if I had time or a social network.

I was asking about financial support to ensure the longevity of the project.
👍️

**Comment by tjyang** [comment]
I agree with @Lockyc, money is one of factors to help this project.  Knowing this project is financially backed by a group of donors is also a plus to gain confidence for new comers.
I like to suggest enabling Sponsors on this github org/repo.  https://github.com/sponsors/SeaQL is an example.

**Comment by kaplanelad** [comment]
Sponsorship for loco is now open—thank you for your past interest, and we'd love to have your support!

**Comment by tjyang** [comment]
Looks like I am 1st sponsor. Self imposed pressure to learn more Rust and Loco ;-)

**Reply by kaplanelad** [reply]
Thank you so much for your support! 🎉 We're thrilled to have you as our first sponsor.
